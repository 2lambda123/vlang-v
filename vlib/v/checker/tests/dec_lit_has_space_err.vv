fn main() {
    a := 123456  789
}
