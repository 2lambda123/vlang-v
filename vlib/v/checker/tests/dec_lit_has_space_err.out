vlib/v/checker/tests/dec_lit_has_space_err.v:2:18: error: unneeded spaces before digit `7`
    1 | fn main() {
    2 |     a := 123456  789
      |                  ^
    3 | }
